client/quantum_serverless/core/job.py

```diff
@@ -127,7 +127,7 @@ def __init__(self, client: JobSubmissionClient):
         self._job_client = client
 
     def status(self, job_id: str):
-        return self._job_client.get_job_status(job_id)
+        return self._job_client.get_job_status(job_id).value
 
```

Comment:
Since `status` is an unimplemented `BaseJobClient` method, I think both the ray and gateway job clients should return a string indicating the status. I updated this in this PR because it affects how I do the status translation in the user-facing job client

Reply:
so, you basically do not want to return status class? and return status string?
I have no objections, just to confirm :)

Reply:
I figured since the gateway job client returns a string, we may as well return a string from the ray job client as well, since they both derive from the same base class. Not a big deal, but I thought it was a clean simplification.